batch/batch/driver/job.py

```diff
@@ -333,13 +343,14 @@ async def make_request():
     log.info(f'unschedule job {id}, attempt {attempt_id}: called delete job')
 
 
-async def job_config(app, record, attempt_id):
+async def job_config(app, record):
```

Comment:
Continuing from https://github.com/hail-is/hail/pull/14170/files#r1476850743

cc: @daniel-goldstein 

Unrelated changes like this are great, *but* they should become separate and independent PRs that are speedily merged completely unencumbered by the PR that spawned them.